client/src/components/CardPool.vue

```diff
@@ -182,6 +171,24 @@ export default {
 			this.columns.pop();
 		},
 	},
+	computed: {
+		shouldCollapse: function() {
+			const r = [].fill(this.columns.length, false);
```

Comment:
```suggestion
			return this.columns.map(column => column.length === 0);
```

Reply:
(per my comment on the PR, if we want to collapse all empty columns, should be really easy (which I imagine you already know)